app/views/items/borrowed_items.html.erb

```diff
@@ -0,0 +1,5 @@
+<p style="color: green"><%= notice %></p>
```

Comment:
- question: What is the purpose of this line? If this is intended for status notifications, such as confirmation of login/logout or of requests, please put it into the layout, because it is then likely to be the same on every page.